This patch is a workaround for a "tail" that should, but doesn't, act like
GNU tail, in that it only reads from one file, even if multiple files are
given on the commandline:

    15273738 tail should behave like head (multiple files)

Since we should fix Solaris tail, this patch is highly Solaris-specific,
and should not be passed upstream.

--- mercurial-3.2.3/tests/test-convert-hg-sink.t	Thu Dec 18 12:59:22 2014
+++ mercurial-3.2.3/tests/test-convert-hg-sink.t	Fri Dec 19 13:03:05 2014
@@ -427,7 +427,7 @@
   $ echo b > b/f
   $ hg -R b ci -mb
 
-  $ tail */.hg/shamap
+  $ gtail */.hg/shamap
   ==> 0/.hg/shamap <==
   86f3f774ffb682bffb5dc3c1d3b3da637cb9a0d6 8a028c7c77f6c7bd6d63bc3f02ca9f779eabf16a
   dd9f218eb91fb857f2a62fe023e1d64a4e7812fe 8a028c7c77f6c7bd6d63bc3f02ca9f779eabf16a
